docs/source/nlp/prompt_learning.rst

```diff
@@ -0,0 +1,378 @@
+Prompt Learning
+-------------
+
+Within NeMo we refer to **p-tuning** and **prompt tuning** methods collectively as prompt learning. Both methods are parameter efficient alternatives to fine-tuning pretrained language models. Our NeMo implementation makes it possible to use one pretrained GPT model on many downstream tasks without needing to tune the model's full set of parameters. It also allows for adding new tasks to your model without overwriting or disrupting previous tasks for which the model has already been p-tuned/prompt-tuned. Because the original model parameters are frozen and never altered by either method, p-tuning/prompt-tuning also avoid cartographic forgetting issues often encountered when fine-tuning models.
+
```

Comment:
maybe you want to emphasize you added this continuous learning capability in the nemo. It is not in the original p-tuning/prompt-tuning paper. It should be one of the selling point for the NeMo.

Reply:
good idea

Reply:
Also, the tutorial notebook has this